### ISSUE TITLE
wish to add the ability to ignore a copy of a property when it has a specific value. For example, null, empty string, empty collection

### ISSUE THREAD

**Issue by WuQuDeRen** [MapsterMapper/Mapster#747]
mapster has provided the function of not copying the property when the value of the property is null. Can we extend this function to stop copying the property when the value of the property is a series of specific values? It is better to provide a general method (I am not sure which property and which type, but I am sure that the value cannot be these)? Determine whether a property field can be copied before it is copied

**Comment by DocSvartz** [comment]
Hello @WuQuDeRen.  Maybe you need [Ignore conditionally](https://github.com/MapsterMapper/Mapster/wiki/Ignoring-members#ignore-conditionally) or [Map with condition](https://github.com/MapsterMapper/Mapster/wiki/Custom-mapping#mapping-with-condition) .

**Comment by WuQuDeRen** [comment]
> Hello @WuQuDeRen. Maybe you need [Ignore conditionally](https://github.com/MapsterMapper/Mapster/wiki/Ignoring-members#ignore-conditionally) or [Map with condition](https://github.com/MapsterMapper/Mapster/wiki/Custom-mapping#mapping-with-condition) .

You're right, but when there are many attributes, configuring them one by one can be cumbersome. I need a universal and concise way
